enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/dependency/ResolveUtil.java

```diff
@@ -66,6 +62,24 @@ class ResolveUtil {
         this.session = Objects.requireNonNull(session);
     }
 
+    /**
+     * Retrieves the {@link DependencyNode} instance containing the result of the transitive dependency
```

Comment:
I'm not sure what "the result of the transitive dependency" is exactly. Is this just the transitive dependencies of the current maven project? Consider rephrasing.

Reply:
Yes it is for current maven project

Reply:
Please rewrite to clarify this

Reply:
please look at next lines - there is:

```
Retrieves the {@link DependencyNode} instance containing the result of the transitive dependency
for the current {@link MavenProject} in verbose mode.
```

Reply:
"the result of the transitive dependency" is unclear. Transitive dependencies are not  process and wouldn't normally be thought of as having results.

Reply:
So ... can you propose description of this method?